iceberg/usecases/injection/src/main/java/com/dremio/iceland/iceberg/injection/SparkProvider.java

```diff
@@ -0,0 +1,24 @@
+package com.dremio.iceland.iceberg.injection;
+
+import org.apache.iceberg.spark.SparkCatalog;
+import org.apache.iceberg.spark.SparkSessionCatalog;
+import org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions;
+import org.apache.spark.sql.SparkSession;
+
+public class SparkProvider {
+
+    public static SparkSession get() {
+        return SparkSession.builder()
+                .master("local[2]")
+                .appName("simple")
+                .config("spark.sql.extensions", IcebergSparkSessionExtensions.class.getName())
+                .config("spark.sql.catalog.spark_catalog", SparkSessionCatalog.class.getName())
```

Comment:
may be we don't need need to configure `spark_catalog`, there is no HMS. So, having a hive catalog is dummy here.